data/scripts/eventcallbacks/party/default_onShareExperience.lua

```diff
@@ -0,0 +1,29 @@
+local ec = EventCallback
+
+ec.onShareExperience = function(self, exp)
```

Comment:
or `function(self, exp, rawExp)`?

Reply:
rawExp is a control variable that is used to know the real experience before any call in the eventcallback system, can be ignored if you don't need it ;)